test/graph_compaction_tests.cc

```diff
@@ -24,7 +27,13 @@ namespace detail {
 	struct region_map_testspy {
 		template <typename T>
 		static size_t get_num_regions(const region_map<T>& map) {
```

Comment:
Note to self: Do we want the number of *distinct values* in the tree, or the number of entries? I suspect the former, might just be the same for these tests by chance.

Reply:
Done!